modules/installation-and-upgrade/pages/proxy-container-installation.adoc

```diff
@@ -118,52 +118,39 @@ The volumes are named:
 
 To override default volume settings, create the volumes prior to the first start of the pod using the  [command]``podman volume create`` command.
 
-It is possible to add custom arguments passed to podman container pod to [path]``/etc/sysconfig/uyuni-proxy-systemd-services.config``:
+To install the systemd service starting the {productname} proxy, run the [command]``mgrpxy install podman /path/to/config.tar.gz`` command.
```

Comment:
We might want to refactor this doc, at the very end, we have a sentence pointing to proxy-container-setup.adoc but maybe we will need it earlier now, as we need the config file at this point on the installation.